### ISSUE TITLE
SIGSEGV

### ISSUE THREAD

**Issue by CameronNemo** [phase1geo/Minder#113]
```
$ gdb com.github.phase1geo.minder
GNU gdb (GDB) 8.3.1
[...]
Reading symbols from com.github.phase1geo.minder...
Reading symbols from /usr/lib/debug//usr/bin/com.github.phase1geo.minder...
(gdb) run
Starting program: /usr/bin/com.github.phase1geo.minder
[Thread debugging using libthread_db enabled]
Using host libthread_db library "/usr/lib/libthread_db.so.1".
[New Thread 0x7ffff3fa1700 (LWP 9400)]
[New Thread 0x7ffff37a0700 (LWP 9401)]
[New Thread 0x7ffff2f9f700 (LWP 9402)]
[New Thread 0x7ffff2786700 (LWP 9403)]

(com.github.phase1geo.minder:9386): GLib-GObject-WARNING **: 21:07:36.265: invalid cast from 'DrawArea' to 'GtkWindow'

(com.github.phase1geo.minder:9386): GLib-GObject-WARNING **: 21:07:36.266: invalid cast from 'DrawArea' to 'GtkWindow'

Thread 1 "com.github.phas" received signal SIGSEGV, Segmentation fault.
0x00007ffff702ac29 in gdk_rgba_to_string (rgba=rgba@entry=0x0) at gdkrgba.c:357
357	gdkrgba.c: No existe el fichero o el directorio.
(gdb) bt
#0  0x00007ffff702ac29 in gdk_rgba_to_string (rgba=rgba@entry=0x0) at gdkrgba.c:357
#1  0x00005555555712e3 in theme_get_css_provider (self=0x555555e6dd60) at com.github.phase1geo.minder@exe/src/Theme.c:678
#2  0x00005555555c9806 in draw_area_load_theme (n=<optimized out>, self=0x555555992530) at com.github.phase1geo.minder@exe/src/DrawArea.c:3225
#3  draw_area_load (n=<optimized out>, self=0x555555992530) at com.github.phase1geo.minder@exe/src/DrawArea.c:3420
#4  document_load (self=<optimized out>) at com.github.phase1geo.minder@exe/src/Document.c:293
#5  0x00005555555d41fd in main_window_load_tab_state (self=0x555555797060) at com.github.phase1geo.minder@exe/src/MainWindow.c:6104
#6  minder_real_activate (base=<optimized out>) at com.github.phase1geo.minder@exe/src/Application.c:303
#7  0x00007ffff7dbcd4f in g_signal_emit_valist () from /usr/lib/libgobject-2.0.so.0
#8  0x00007ffff7dbd9ff in g_signal_emit () from /usr/lib/libgobject-2.0.so.0
#9  0x00007ffff6e4cd08 in ?? () from /usr/lib/libgio-2.0.so.0
#10 0x00007ffff6e4c87a in g_application_run () from /usr/lib/libgio-2.0.so.0
#11 0x0000555555567ff6 in minder_main (args_length1=<optimized out>, args=<optimized out>) at com.github.phase1geo.minder@exe/src/Application.c:523
#12 main (argc=<optimized out>, argv=<optimized out>) at com.github.phase1geo.minder@exe/src/Application.c:534
```

gtk version: 3.24.13

**Comment by phase1geo** [comment]
Would it be possible to see the contents of the Minder file that you are attempting to load? If there are any concerns of sensitive information in the file, I'm really just interested in the portions of the file that are encapsulated within the <theme>...</theme> sections which should be at the top of the file.

**Comment by phase1geo** [comment]
By the way, thanks for including the gdb output. This really helps debug the issue much quicker.

**Comment by CameronNemo** [comment]
Here is the minder file:

```xml
<?xml version="1.0"?>
<minder>
  <theme name="Predeterminado" index="1"/>
  <styles>
    <style level="0" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="rounded" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
    <style level="1" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
    <style level="2" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
    <style level="3" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
    <style level="4" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
    <style level="5" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
    <style level="6" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
    <style level="7" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
    <style level="8" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
    <style level="9" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
    <style level="10" linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true" connectiondash="dotted" connectionwidth="2" connectionarrow="fromto" connectionpadding="3" connectionfont="Sans 10"/>
  </styles>
  <drawarea x="703.18749999999966" y="17.96484375" scale="1"/>
  <images>
    <image id="3" uri="file:///home/cameronnemo/media/fotos/fam2017people/23758013_1476015959178922_1674331223_o.jpg" ext=".jpg"/>
  </images>
  <nodes>
    <node id="1" posx="-335.18749999999966" posy="386.03515625" maxwidth="200" width="130" height="46" side="left" fold="false" treesize="453" layout="Horizontal">
      <style linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true"/>
      <nodename>Idea principal</nodename>
      <nodenote></nodenote>
      <nodes>
        <node id="3" posx="-105.18749999999989" posy="386.03515625" maxwidth="200" width="80" height="46" side="right" fold="false" treesize="453" color="#c6262e" layout="Horizontal">
          <style linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true"/>
          <nodename>ckjdskj</nodename>
          <nodenote></nodenote>
          <nodes>
            <node id="5" posx="74.812500000000114" posy="182.53515625" maxwidth="200" width="228" height="407" side="right" fold="false" treesize="407" color="#c6262e" layout="Horizontal">
              <nodeimage id="3" x="0" y="0" w="338" h="600"/>
              <style linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true"/>
              <nodename>dkjkdsf</nodename>
              <nodenote></nodenote>
            </node>
            <node id="7" posx="74.812500000000114" posy="589.53515625" maxwidth="200" width="86" height="46" side="right" fold="false" treesize="46" color="#c6262e" layout="Horizontal">
              <style linktype="straight" linkwidth="4" linkarrow="false" linkdash="solid" nodeborder="underlined" nodewidth="200" nodeborderwidth="4" nodefill="false" nodemargin="8" nodepadding="6" nodefont="Sans 11" nodemarkup="true"/>
              <nodename>dskjdsk</nodename>
              <nodenote></nodenote>
            </node>
          </nodes>
        </node>
      </nodes>
    </node>
  </nodes>
</minder>
```

**Comment by phase1geo** [comment]
Hmmm. That theme line is not correct at all. If you want to remove that line from the .minder file, you should be able to run again. I’ll look into how this could be happening.

**Comment by phase1geo** [comment]
This bug should now be fixed in the development branch and will be generally available in the next stable release.

**Comment by phase1geo** [comment]
This bug has been fixed in the 1.7.0 stable release.  Closing bug.
